week2/index.js

```diff
@@ -58,6 +101,78 @@ function listTodos() {
   });
 }
 
+//add items to todo list
+function add() {
+  //check if something typed after add
+  if (options[1] == null) {
+    console.log(`type somthing after add `);
+    return;
+  }
+  let item = options.map(i => {
+    //the command(add) not to be included
+    if (i !== command)
+      return i
+  }).join(' ').trim()
+  fs.appendFile('todo.txt', `\r\n${item}`, (err) => {
+    if (err) throw err;
+    console.log(`${item} was appended to file!`);
+  });
+}
+
+
+//remove items from todo list
+function remove() {
+  fs.readFile('todo.txt', 'utf8', function(err, data) {
+    if (err) {
+      throw err;
+    }
+    //parse string to number
+    let number = parseInt(options[1])
+    let lines = splitStringByNewline(data);
+    //avoid type anything but numbers
+    if ((typeof number === 'number' && !isNaN(number))) {
+      // number has not to be greater than list number
+      if (number > lines.length) {
+        console.log(`there is ${lines.length} lines in this file choose between 0 and ${lines.length} `);
+      }
+      //after remove should not be empty
+    } else if (options[1] == null) {
+      console.log(`type a number after remove `);
+      return;
+    } else {
+      console.log('use only numbers');
+      return
+    }
+    lines.splice(number - 1, 1);
+    let joinedLines = lines.join('\n').trim();
+    fs.writeFile('todo.txt', joinedLines);
+
+  });
+}
+
+
+//remove all todo items from the list
+function reset() {
+
+  fs.truncate('todo.txt', 0, function() {
+    console.log('"todo" list is empty now!!')
+  })
+}
+
+//update a todo item with new text
+function update() {
+  fs.readFile('todo.txt', 'utf8', function(err, data) {
+    if (err) {
+      throw err;
+    }
+    todo = splitStringByNewline(data)
+    let number = num()
+    todo[number - 1] = options[2]
+    let joinedLines = todo.join('\r\n').trim();
+    fs.writeFile('todo.txt', joinedLines);
```

Comment:
You mean "fs.writeFileSync"? When I run your code I get:
`(node:1614) [DEP0013] DeprecationWarning: Calling an asynchronous function without callback is deprecated.`